src/main/java/ch/heigvd/dai/utilities/FileManipulator.java

```diff
@@ -0,0 +1,56 @@
+package ch.heigvd.dai.utilities;
```

Comment:
Missing header / copyright

Reply:
This is already being done in [this branch](https://github.com/lentidas/DAI-2425-PW1/tree/docs/add_readme_documentation).